pstress/repeat_crash.sh

```diff
@@ -0,0 +1,227 @@
+############################################################################################
+# Created by Mohit Joshi, Percona LLC                                                      #
+# Creation date: 07-JAN-2021                                                               #
+#                                                                                          #
+# This script is created to repeat crashes found during pstress runs.                      #
+# The script requires the incident directory, the mysql basedir, mode and the nth step     #
+# from where the pstress runs will be resumed to repeat crashes.                           #
+# The script requires to be run from <source>/pstress directory                            #
+# Usage: ./repeat_crash.sh -i$HOME/path/to/incident/dir -b/path/to/basedir -s3 -mcurrent   #
+############################################################################################
+
+#!/bin/bash
+
+Help()
+{
+echo "Syntax: repeat_crash [-i|s|m|b]";
+echo "options:
+-i	Path to the Incident Directory where pstress was executed
+-s	Nth step where the crash is seen
```

Comment:
The script will work for all three scenarios i.e kill,sig4 and shutdown. It depends on the value of SIGNAL
If there is some other problem other than crash, it will be detected by the search_string.sh .